common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_bml.yaml

```diff
@@ -498,11 +498,14 @@ cACTION_BML_CLIENT_GET_CLIENT_LIST_REQUEST:
 
 cACTION_BML_CLIENT_GET_CLIENT_LIST_RESPONSE:
   _type: class
+  result:
+    _type: uint8_t
+    _comment: # 0 - Failure, 1 - Success
```

Comment:
you are correct - added the following description to the commit:
```
Added a missing `result` parameter to the response message to indicate
an error when needed (since there are successful cases of an empty 
list).
```